policy.md

```diff
@@ -1,178 +1,116 @@
+4. List of the CA policy documents (with version numbers) referenced during the audit;
+5. Whether the audit is for a period of time or a point in time;
+6. Date the audit statement was written, which will necessarily be after the audit period end date or point-in-time date (see date format specifications below);
+7. Start date and end date of the period that was audited, for those that cover a period of time (this is not the period the auditor was on-site);
+8. Point-in-time date, for those that are for a point in time;
+9. Full names and version numbers of the audit standards that were used during the audit; 
+10. All incidents disclosed by the CA, discovered by an auditor, or reported by a third party, that, at any time during the audit period, occurred, were open in Bugzilla, or were reported to a Store; and
+11. For ETSI, a statement to indicate if the audit was a full audit, and which parts of the criteria were applied, e.g. DVCP, OVCP, NCP, NCP+, LCP, EVCP, EVCP+, QCP-w, Part1 (General Requirements), and/or Part 2 (Requirements for trust service providers).
 
-Audits based on ETSI CPs: Audits conducted by accredited conformity assessment bodies (CAB) must have their Audit Attestation Letter (AAL) uploaded to the CAB’s website. CAs provide the URL to the AAL on the CAB’s website, and ALV will verify those URLs against a known list of AAL locations.
+#### 5.1.1 ETSI 
 
-* When an ETSI Certificate cannot be issued, the CA must still provide an AAL such that there are no gaps between audit periods for consecutive audits. The CA may post the AAL on their own website or attach the attestation report to a [Bugzilla Bug](https://www.ccadb.org/cas/fields#uploading-documents) and provide that URL. Additionally, the CA needs to provide an explanation about the problems and time frame for resolution of the problems.
+Audits conducted by accredited CAB must have their AAL uploaded to the CAB’s website. CAs provide the URL to the AAL on the CAB’s website, and ALV will verify those URLs against a known list of AAL locations.
 
-WebTrust Audits: Audits conducted by licensed WebTrust practitioners must have a WebTrust Seal. CAs enter the URL to the WebTrust Seal into the CCADB, and upon saving of the record, the CCADB automatically converts the URL to point to the corresponding PDF file via integration with CPA Canada.
+When an ETSI Certificate cannot be issued, the CA must still provide an AAL such that there are no gaps between audit periods for consecutive audits. The CA may post the AAL on their own website or attach the attestation report to a [Bugzilla Bug](https://www.ccadb.org/cas/fields#uploading-documents) and provide that URL. Additionally, the CA must provide an incident report for each audit finding (i.e., non-conformity),  and/or problem resulting in the failure of the ETSI Certificate’s issuance, in a Bugzilla Bug within seven calendar days of the audit attestation letter’s issuance date.
```

Comment:
New requirement for CCADB SC discussion: "Additionally, the CA must provide an incident report for each audit finding (i.e., non-conformity), and/or problem resulting in the failure of the ETSI Certificate’s issuance, in a Bugzilla Bug within seven calendar days of the audit attestation letter’s issuance date."